CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 - Updates the `FeedbackBox` component to remove the underline on the component's `Privacy Policy` link.
 - Updates `DatePicker` to pass a `additionalHelperTextIds` to its `TextInput` if needed so that the `aria-describedby` value can be associated with all relevant `helperText`s.
 - Adds the `FeaturedContent` component.
+- Updates the `FeedbackBox` and `NewsletterSignup` components to set the `tabindex` on the internal elements to `"-1"`.
```

Comment:
This will be good to add to each component in their changelog data table section.

Reply:
I did not add anything to the changelog for `NewsletterSignup` becuase this code is part of the new component.  It's not a change.  However, I did update the Accessibility docs to talk about the -1 tabindex value.